README.md

```diff
@@ -123,7 +142,7 @@ The Event Management Agent comes with the following event or message broker plug
 The default application.yml provides various plugin examples. For Kafka, the properties section under credentials is
 passthrough. For example a property in ConsumerConfig or SSLConfigs classes.
 
-If using AWS IAM, the AWS Access Key Id and AWS Secret Access Key need to be present. Two ways is either via environment
+If using AWS IAM, the AWS Access Key Id and AWS Secret Access Key need to be present. This can be done via environment
```

Comment:
Nitpick - "This can be done via environment or credentials file as shown below:"
The statement indicates environment, then creds file. Then in the examples, credentials file is shown first, followed by environment configuration. The order of examples should match the statement.

Reply:
Done